templates/qpidd.conf.erb

```diff
@@ -44,3 +44,6 @@ interface=<%= scope['qpid::interface'] %>
 <% unless [nil, :undefined, :undef, ''].include?(scope['qpid::max_connections']) -%>
 max-connections=<%= scope['qpid::max_connections'] %>
 <% end %>
+<% unless [nil, :undefined, :undef, ''].include?(scope['qpid::params::session_unacked']) -%>
```

Comment:
This should be `qpid::session_acked`, not the value in qpid::params.  Params is a pattern to hold default values, it'll always be undef.